apps/algolia/lib/algolia/update.ex

```diff
@@ -32,10 +61,28 @@ defmodule Algolia.Update do
     |> do_has_routes?(data)
   end
 
+  defp current_objects(base_url, index_module) do
+    with {:ok, body} <- get_current_objects(base_url, index_module) do
+      response_to_objects(body)
```

Comment:
Looks like we'll need to do some additional processing here (or in `response_to_objects/1`) to get _all_ the objects.

The results returned from Algolia are paginated - so, requesting the stops returns 50 pages of results, each listing 20 hits. This can be read directly from the response object, which includes in addition to `hits`:

```
"nbHits": 8074,
"page": 0,
"nbPages": 50,
"hitsPerPage": 20,
```

Navigating this is handled by query parameters, e.g. `hitsPerPage=1000` can be used to change the default (1000 is the max) and `&page=3` will go to page 3 of the results.

Reply:
Are you sure? It returns all the records but is still paginated. Improvement - it appears to automatically set it to the max page size.

```
"nbHits": 8074,
"page": 0,
"nbPages": 9,
"hitsPerPage": 1000,
```